src/Orleans/MultiCluster/IProtocolParticipant.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Threading.Tasks;
+using Orleans.Concurrency;
+
+namespace Orleans.MultiCluster
+{
+    /// <summary>
+    /// Grain interface for grains that participate in multi-cluster-protocols.
+    /// </summary>
+    public interface IProtocolParticipant  : IGrain  
```

Comment:
I could not find any implementation of `IProtocolParticipant`. Are you sure it belongs to this PR? Or should we add it when we actually have an implementation of it? (I assume the global single instance requires one or is it for queued grain?)